desktop/core/src/desktop/lib/django_util.py

```diff
@@ -26,6 +26,7 @@
 
 from django.conf import settings
 from django.core import serializers
+from django.core.exceptions import FieldDoesNotExist
```

Comment:
@romainr  please check it also.

Reply:
Will it brake older Django?

Reply:
1- > You can see the from `django.core.exceptions import FieldDoesNotExist  # NOQA` @ https://docs.djangoproject.com/en/1.11/_modules/django/db/models/fields/

2-> https://docs.djangoproject.com/en/3.1/releases/3.1/#id1 at this link we see that `The compatibility import of django.core.exceptions.FieldDoesNotExist in django.db.models.fields is removed.`

3-> And in the last one we can see that source of `FieldDoesNotExist` for both Django ( 1 and 3) is same 
        1->https://docs.djangoproject.com/en/3.1/_modules/django/core/exceptions/#FieldDoesNotExist (for Django 3.1)
        2->https://docs.djangoproject.com/en/1.11/_modules/django/core/exceptions/#FieldDoesNotExist (for Django 1.11)

Reply:
:+1: Perfect, very clear!